runtime/j9vm/jvm.c

```diff
@@ -3577,69 +3577,25 @@ JVM_LoadSystemLibrary(const char *libName)
  *	DLL: jvm
  */
 
-/* NOTE THIS IS NOT REQUIRED FOR 1.4 */
+/* NOTE THIS IS NOT REQUIRED FOR 1.4
```

Comment:
This line can be removed since OpenJ9 doesn't support Java 1.4

Reply:
Removed.